scripts/run-integration-tests.sh

```diff
@@ -40,7 +40,7 @@ main() {
         echo "Downloading Fabric Branch v1.4.0"
         git clone https://github.com/hyperledger/fabric $FABRIC_DIR --branch v1.4.0 --single-branch --depth 1
     else
-        FABRIC_VERSION=$(git -C ${FABRIC_DIR} describe)
```

Comment:
I have some patches on 1.4.0 that helps it build better, this allows that, but not 1.4.1 or anything like that.

Maybe there's a better way to do it?

Reply:
We don't want to allow 1.4.1 or others necessarily. For example 1.4.3 has breaking changes which doesn't affect fab3 or evmcc, but it does affect our integration tests, hence the need to be strict about the version.

Reply:
It does not allow 1.4.1 if the commit for 1.4.1 is tagged.

Reply:
if there's a breaking change, [shouldn't the minor version be bumped, not the patch](https://semver.org)?

Reply:
I agree about semvar. 1.4.3 has some new required keys in the configuration, which cause our integration tests to fail because we are using the keys from 1.4.0.

Reply:
I don't understand why we're having this discussion.